### PR TITLE
Field focus wrapper and toolbar design update

### PR DESCRIPTION
**Pull request by csAyushDubey**
Ref: [VE-3781](https://contentstack.atlassian.net/browse/VE-3781)

[VE-3781]: https://contentstack.atlassian.net/browse/VE-3781?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/visualBuilder/generators/generateToolbar.tsx [hunk 1]
```diff
@@ -34,12 +31,10 @@ export function appendFieldToolbar(
...
     )
         return;
-    const { editableElement, fieldMetadata } = eventDetails;
     const wrapper = document.createDocumentFragment();
     render(
         <FieldToolbarComponent
-            fieldMetadata={fieldMetadata}
-            editableElement={editableElement}
+            eventDetails={eventDetails}
         />,
```
Comment (hiteshshetty-dev) [inline src/visualBuilder/generators/generateToolbar.tsx:38]: Do we need this changes? Any particular reason for it?

Comment (csAyushDubey) [reply src/visualBuilder/generators/generateToolbar.tsx:38]: we need `cslpData` of eventDetails, for open quick form post message event

**Review by hiteshshetty-dev — APPROVED** [review]
LGTM!!

**Review by hiteshshetty-dev — APPROVED** [review]
LGTM!!

**Pull request merged**